mantaray/backend.py

```diff
@@ -503,6 +503,19 @@ def send_privmsg(self, nick_or_channel: str, text: str) -> None:
         )
 
     # emits SelfChangedNick event on success
+
+    def kick(self, channel: str, nick: str, reason: str | None = None) -> None:
+        if reason is None:
+            reason=''
```

Comment:
I don't think this is enough. With no reason, it should send `KICK #lol Akuli`, not `KICK #lol Akuli :`